posthog/api/integration.py

```diff
@@ -94,3 +100,34 @@ def channels(self, request: Request, *args: Any, **kwargs: Any) -> Response:
         ]
 
         return Response({"channels": channels})
+
+    @action(methods=["GET"], detail=True, url_path="conversionActions")
+    def conversion_actions(self, request: Request, *args: Any, **kwargs: Any) -> Response:
```

Comment:
Can we make this url path `google_conversion_actions` or something really specific. The `channels` one should really have been `slack_channels`

Reply:
done - [`efcda05` (#26088)](https://github.com/PostHog/posthog/pull/26088/commits/efcda05b6659b0be4c8636ab7f2164dd12356b31)